update note (#4120)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md

title: Work with Okta session cookies
---

> > **Note:** This document is written for Okta Classic Engine. If you are using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and the [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section of the Identity Engine limitations page. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.

Reviewer: could use "you're" to be more conversational

> > **Note:** This document is written for Okta Classic Engine. If you are using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and the [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section of the Identity Engine limitations page. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.

Author: ```suggestion
> **Note:** This document is written for Okta Classic Engine. If you're using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and the [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section of the Identity Engine limitations page. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```
